spec/models/o_auth_application_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
This is probably failing the build, no idea why it would ever work. Just drop the file for now and recreate it properly once you add specs.